src/main/java/ch/njol/skript/expressions/ExprNearestEntity.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * This file is part of Skript.
+ *
+ * Skript is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Skript is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.entity.EntityData;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import ch.njol.util.StringUtils;
+import org.bukkit.Location;
+import org.bukkit.entity.Entity;
+import ch.njol.skript.lang.util.SimpleExpression;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.Arrays;
+
+@Name("Nearest Entity")
+@Description("Gets the entity nearest to a location or another entity.")
+@Examples({"kill the nearest pig and cow relative to player",
+	"teleport player to the nearest cow relative to player",
+	"teleport player to the nearest entity relative to player",
+	"on click:\n\tkill nearest pig"})
```

Comment:
```suggestion
	"",
	"on click:",
	"\tkill nearest pig"})
```
Avoiding the use of `\n`. The empty line above just makes it look clearer imo
Also, Walrus suggested this style which looks better
```java
@Examples({
    "kill the nearest pig and cow relative to player",
	"teleport player to the nearest cow relative to player",
	"teleport player to the nearest entity relative to player",
    "",
	"on click:",
    "\tkill nearest pig"
})
```